### PR TITLE
Use the imported statement instead of FQN

### PR DESCRIPTION
**Pull request by BackEndTea**
Meaning we dont need the phan-suppress either

### CODE REVIEW

File: lib/Compiler.php [hunk 1]
```diff
@@ -10,7 +10,6 @@
 namespace PHPCompiler;
 
 // This is used as a property type, phan is confused.
-// @phan-suppress-next-line PhanUnreferencedUseNormal
```
Comment (driusan) [inline lib/Compiler.php]: should probably remove the comment above explaining why it was suppressed too.

File: lib/Compiler.php [hunk 2]
```diff
@@ -10,7 +10,6 @@
 namespace PHPCompiler;
 
 // This is used as a property type, phan is confused.
```
Comment (driusan) [inline lib/Compiler.php]:
```suggestion
```

Just realized I could do my comment as a github suggestion..

Comment (BackEndTea) [reply lib/Compiler.php]: Never used the github suggestions to make a commit, so that was a first.

**Comment by BackEndTea** [comment]
Thank you @ircmaxell & @driusan

**Comment by ircmaxell** [comment]
Thank You!!!

**Pull request merged**